src/wallet/wallet.cpp

```diff
@@ -2149,7 +2149,7 @@ bool CWallet::SelectCoinsMinConf(const CAmount& nTargetValue, const int nConfMin
     pair<CAmount, pair<const CWalletTx*,unsigned int> > coinLowestLarger;
     coinLowestLarger.first = std::numeric_limits<CAmount>::max();
     coinLowestLarger.second.first = NULL;
-    vector<pair<CAmount, pair<const CWalletTx*,unsigned int> > > vValue; // List of values less than target
+    std::vector<pair<CAmount, pair<const CWalletTx*,unsigned int> > > vValue; // List of values less than target
```

Comment:
This is already done (for the entire file) by @kallewoof in https://github.com/bitcoin/bitcoin/pull/9643